.github/workflows/notion.yml

```
@@ -0,0 +1,101 @@
+  pull_request:
+    types: [opened, closed]
+
+jobs:
+  update-notion:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Extract PBI Number from PR Title
+        run: |
```

actionlint (1.7.4)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)